src/pollution_model/run_pollution_model.py

```
@@ -0,0 +1,454 @@
+        total_zinc_load = total_zinc_load * factor + second_stage_zinc * surface_area * bias_factor
+        total_copper_load *= total_copper_load * factor + second_stage_copper * surface_area * bias_factor
+
+    return total_copper_load, total_zinc_load
+
+
+def total_metal_load_road_carpark(tss_surface):
+    """
+    Calculate the total metal load for car parks and roads from their total suspended solids.
+
+    Parameters
+    ----------
+    tss_surface: float
+        total suspended solids of this surface
+
+    Returns
+    -------
+    float, float
+       Returns the total copper and zinc loads for this surface
+    """
+    # Define constants
+    d = 0.441
+    e = 1.96
+    # Return total copper load, total zinc load
+    return tss_surface * d, tss_surface * e
+
+
+def dissolved_metal_load(total_copper_load, total_zinc_load, surface_type):
+    """
+        Calculate the dissolved metal load for all surfaces from their total suspended solids.
+
+        Parameters
+        ----------
+        total_copper_load: float
+            total copper load for the surface
+        total_zinc_load: float
+            total zinc load for the surface
+        surface_type: int
+            The type of surface that we are calculating this for
+
+        Returns
+        -------
+        float, float
+           Returns the dissolved copper and zinc loads for this surface
+        """
+    # Declare constants
+    f = 1
+    g = 1
+    # Set constant values based on surface type
+    match surface_type:
+        case 1:
+            f = 0.46
+            g = 0.67
+        case 2:
+            f = 0.77
+            g = 0.72
+        case 3:
+            f = 0.28
+            g = 0.43
+        case 4:
+            f = 0.28
+            g = 0.43
+        case _:
+            log.error(
+                f"Given surface type is not valid for computing dissolved metal load. {SurfaceType(surface_type).name}.")

```

Should raise an error, rather than just log it, so that we can propagate the errors to the front-end